lib/DDG/Rewrite.pm

```diff
@@ -131,6 +136,7 @@ sub _build_nginx_conf {
 	my $wrap_jsonp_callback = $self->has_callback && $self->wrap_jsonp_callback;
 	my $wrap_string_callback = $self->has_callback && $self->wrap_string_callback;
 	my $uses_echo_module = $wrap_jsonp_callback || $wrap_string_callback;
+        my $callback = $self->has_callback ? $self->callback : '';
```

Comment:
Yeah, exactly.  I think you can just do `my $callback = $self->callback`.  Only IAs that [support this behavior](https://github.com/duckduckgo/duckduckgo/blob/master/lib/DDG/Meta/ZeroClickInfoSpice.pm#L175) should get here.  Other than that small change and the spacing irregularities, lgtm.